<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PlayerControls</class>
    <widget class="QFrame" name="PlayerControls">
        <property name="styleSheet">
            <string notr="true">
                PlayerControls {
                background-color: rgb(23, 24, 27);
                }
                Icon{
                width: 60px;
                min-width: 60px;
                height: 60px;
                min-height: 60px;
                color: rgb(204, 204, 204);
                border: 2px solid rgb(204, 204, 204);
                border-radius: 10;
                font-size: 26px;
                }
                Icon:focus{
                color: rgb(255, 255, 255);
                background-color: rgba(45, 114, 217, 0.6);
                border-color: rgb(255, 255, 255);
                }
                TimeLabel{
                color: rgb(255, 255, 255);
                font-family: "Open Sans Semibold";
                font-size: 24px;
                }
            </string>
        </property>
        <layout class="QGridLayout" name="controlsLayout">
            <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="leftMargin">
                <number>20</number>
            </property>
            <property name="topMargin">
                <number>15</number>
            </property>
            <property name="rightMargin">
                <number>20</number>
            </property>
            <property name="bottomMargin">
                <number>15</number>
            </property>
            <item row="0" column="0">
                <layout class="QGridLayout">
                    <property name="verticalSpacing">
                        <number>10</number>
                    </property>
                    <property name="horizontalSpacing">
                        <number>20</number>
                    </property>
                    <item row="0" column="0">
                        <widget class="Icon" name="stopButton">
                            <property name="maximumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="Icon" name="backwardButton">
                            <property name="maximumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="2">
                        <widget class="Icon" name="playPauseButton">
                            <property name="maximumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="3">
                        <widget class="Icon" name="forwardButton">
                            <property name="maximumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="4">
                        <widget class="Icon" name="moreButton">
                            <property name="maximumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="TimeLabel" name="time">
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1" colspan="3">
                        <widget class="ProgressControl" name="progress">
                        </widget>
                    </item>
                    <item row="1" column="4">
                        <widget class="TimeLabel" name="duration">
                            <property name="alignment">
                                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>TimeLabel</class>
            <extends>QLabel</extends>
            <header>widgets/TimeLabel.h</header>
        </customwidget>
        <customwidget>
            <class>Icon</class>
            <extends>QLabel</extends>
            <header>widgets/Icon.h</header>
        </customwidget>
        <customwidget>
            <class>ProgressControl</class>
            <extends>QFrame</extends>
            <header>widgets/ProgressControl.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
